### DISCUSSION TITLE
Does not display webcam result

### DISCUSSION THREAD

**Discussion by krolaper** [ultralytics/yolov5#10025] [Q&A] [answered]
Tell me, all the parameters are standard, but it does not display the result of the webcam and stream video.
Tried to set view-img True - didn't help

**Comment by krolaper** [comment, marked as answer]
Saw that there was a fix to this problem, updated the code.
I’m trying to: `detect.py --source 0`
Hangs in the stage:
```
YOLOv5 🚀 2022-11-6 Python-3.10.6 torch-1.12.1+cu102 CUDA:0 (NVIDIA GeForce MX130, 2004MiB)

Fusing layers... 
YOLOv5m summary: 290 layers, 22323858 parameters, 0 gradients
```

**Reply by glenn-jocher** [reply]
@krolaper webcam inference works correctly for me. If you can produce an error/issue in a common environment and a common source like YouTube in Colab please let us know, otherwise there is no action for us to take.

<img width="1470" alt="Screenshot 2022-11-06 at 17 26 34" src="https://user-images.githubusercontent.com/26833433/200182557-7423bdda-89ba-488b-a127-dd948327fd43.png">

**Comment by krolaper** [comment]
Thank you. Reinstalled yolo and it worked. When you close the webcam window, it reboots. Where to fix it to close?
